scripts/index.js

```diff
@@ -56,6 +56,14 @@ function populateBook() {
   });
 }
 
+function loadFromStorage() {
+  if (storageAvailable('localStorage')) {
+    const { localStorage } = window;
+    bookList = JSON.parse(localStorage.getItem('bookList'));
+    populateBook();
+  }
+}
```

Comment:
- [ ] Good job so far :+1: but right now I cannot even add a book due to this console error and as a result, I cannot verify if the local storage works. Kindly fix this issue and make sure that your app works as expected :100: 
- Console error:
![Screenshot from 2022-08-09 08-02-29](https://user-images.githubusercontent.com/85113567/183586073-8dd5cacf-c6b6-4548-91fe-ef8f1a07e0dd.png)